weblate/accounts/notifications.py

```diff
@@ -622,6 +622,15 @@ class TranslatedStringNotificaton(Notification):
     filter_languages = True
 
 
+@register_notification
+class TranslatedStringRemovedNotificaton(Notification):
+    actions = (Change.ACTION_STRING_REMOVE,)
+    # Translators: Notification name
+    verbose = _("Translated string removed")
+    template_name = "translated_string_removed"
+    filter_languages = True
```

Comment:
I don't see a use case for this notification - not all change events need a notification e-mail.

Reply:
Fair point. I have removed the email notification and associated html files.